components/code_block/__snapshots__/code_block.test.tsx.snap

```diff
@@ -115,9 +115,9 @@ exports[`codeBlock should render html code block with proper indentation 1`] = `
         dangerouslySetInnerHTML={
           Object {
             "__html": "\`\`\`html
-<span class=\\"hljs-tag\\">&lt;<span class=\\"hljs-name\\">div</span> <span class=\\"hljs-attr\\">className</span>=<span class=\\"hljs-string\\">&#x27;myClass&#x27;</span>&gt;</span>
-  <span class=\\"hljs-tag\\">&lt;<span class=\\"hljs-name\\">a</span> <span class=\\"hljs-attr\\">href</span>=<span class=\\"hljs-string\\">&#x27;https://randomgibberishurl.com&#x27;</span>&gt;</span>ClickMe<span class=\\"hljs-tag\\">&lt;/<span class=\\"hljs-name\\">a</span>&gt;</span>
-<span class=\\"hljs-tag\\">&lt;/<span class=\\"hljs-name\\">div</span>&gt;</span>
+&lt;div className=&apos;myClass&apos;&gt;
```

Comment:
Would it be possible to add a test to `code_block.test.tsx` that renders these both before and after loading the library? Ideally, this snapshot would look the same once the library is loaded.

Reply:
I added test to take snapshot before and after syntax highlighting.